cpp/meson.build

```diff
@@ -0,0 +1,19 @@
+project('Lift-Kata', 'cpp', default_options : ['cpp_std=c++17'])
```

Comment:
I see that C++17 is mentioned here - Maybe I missed it, but I didn't see where the C++ version was specified in CMake...

As users could build the same source code via two different build systems, I wonder whether it would be worth making them use a consistent C++ standard - and also, since I was able to build with C++11 and cmake, whether it needs to be as high as 17, or whether an earlier version would open this kata up to more users?

Reply:
I downgraded to C++-11, but missed the meson file, again :)

```cmake
add_executable(liftsystem_catch2
        src/lift.h
        src/lift.cpp
        test/lift_catch.cpp
        test/liftsystem_catch.cpp
        test/main.cpp)
set_target_properties(liftsystem_catch2 PROPERTIES CXX_STANDARD 11)
```

Reply:
Thanks - I was looking in the wrong place for the version in CMake - it's great that it's 11!